README.md

```diff
@@ -3,3 +3,60 @@
 This repo is used to collaboratively build user documentation.  Pull Requests welcome!  View specific topics below.
 
 ### [Uploading shape data](uploads.md)
+
+## Permissions
+
+### Labs
+
+| Action                                                                                      | Public | Viewer | Editor | Admin |
+|---------------------------------------------------------------------------------------------|--------|--------|--------|-------|
+| Access to public labs                                                                       |   ✅   |   ✅   |   ✅   |   ✅   |
+| View public scenarios                                                                       |   ✅   |   ✅   |   ✅   |   ✅   |
+| Modify unlocked global inputs in unlocked <br> global scenarios without app storing values  |   ✅   |   ✅   |   ✅   |   ✅   |
+| About modal                                                                                 |   ❌   |   ✅   |   ✅   |   ✅   |
+| Simulations dropdown                                                                        |   ❌   |   ✅   |   ✅   |   ✅   |
+| Download sims                                                                               |   ❌   |   ✅   |   ✅   |   ✅   |
+| Scenarios dropdown                                                                          |   ❌   |   ✅   |   ✅   |   ✅   |
+| Bookmarks panel                                                                             |   ❌   |   ❌   |   ✅   |   ✅   |
+| Create scenarios                                                                            |   ❌   |   ❌   |   ✅   |   ✅   |
+| Edit scenarios                                                                              |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete scenarios                                                                            |   ❌   |   ❌   |   ✅   |   ✅   |
+| Duplicate scenarios                                                                         |   ❌   |   ❌   |   ✅   |   ✅   |
+| Run sims                                                                                    |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete sims                                                                                 |   ❌   |   ❌   |   ✅   |   ✅   |
```

Comment:
Would be helpful to distinguish whether they can delete ANY sim or just those that they created.

Reply:
According to `isEditable: authStore.isAdmin || authStore.isEditor || creatorUid === authStore.uid,` admins and editors always can edit/delete and any other user only can when is the creator. Is that correct, @fscottfoti ?

Reply:
I think this is a question for @josh-p-thompson - that's a bug right?